sites/partners/page_content/locale_overrides/general.json

```diff
@@ -33,7 +33,7 @@
   "application.details.householdSize": "Household Size",
   "application.details.language": "Application Language",
   "application.details.monthlyIncome": "Monthly Income",
-  "application.details.number": "Application Number",
+  "application.details.number": "Confirmation Code",
```

Comment:
We were calling the confirmation code "Application Number" which I worried was confusing especially given the latest lottery

Reply:
I think we may want to check in with PM/design (if you hadn't already) about this change

Reply:
Yep yep scheduled a message for Em tmrw :)